internal/core/analysis_api/handlers/list_globals.go

```diff
@@ -0,0 +1,145 @@
+package handlers
+
+/**
+ * Panther is a Cloud-Native SIEM for the Modern Security Team.
+ * Copyright (C) 2020 Panther Labs Inc
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import (
+	"errors"
+	"net/http"
+	"sort"
+	"strconv"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/expression"
+
+	"github.com/panther-labs/panther/api/gateway/analysis/models"
+	"github.com/panther-labs/panther/pkg/gatewayapi"
+)
+
+// ListGlobals pages through globals from a single organization.
+func ListGlobals(request *events.APIGatewayProxyRequest) *events.APIGatewayProxyResponse {
+	var err error
+
+	// Parse the input
+	ascending := defaultSortAscending
+	if sortDir := request.QueryStringParameters["sortDir"]; sortDir != "" {
+		switch sortDir {
+		case "ascending":
+			ascending = true
+		case "descending":
+			ascending = false
+		}
```

Comment:
perhaps more simply,

```suggestion
		ascending = sortDir == "ascending"
```

Reply:
Feels very pythonic to me, gives me shivers. updated.

Reply:
Haha what is Go but a faster and safer Pythonic language?